Description: Relax dependencies in vec-collections
 Relaxing dependencies in vec-collections
Author: Emmanuel Arias <eamanu@debian.org>
Last-Update: 2024-09-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- vec-collections.orig/Cargo.toml
+++ vec-collections/Cargo.toml
@@ -38,28 +38,28 @@
 name = "set_ops"
 harness = false
 [dependencies.binary-merge]
-version = "0.1.1"
+version = "0.1"
 
 [dependencies.bytecheck]
-version = "0.6.5"
+version = "0.6"
 optional = true
 
 [dependencies.inplace-vec-builder]
-version = "0.1.0"
+version = "0.1"
 features = ["smallvec"]
 
 [dependencies.lazy_static]
-version = "1.4.0"
+version = "1.4"
 
 [dependencies.num-traits]
 version = "0.2"
 
 [dependencies.parking_lot]
-version = "0.11.2"
+version = "0.12"
 optional = true
 
 [dependencies.rkyv]
-version = "0.7.18"
+version = "0.7"
 optional = true
 
 [dependencies.serde]
@@ -68,15 +68,12 @@
 default-features = false
 
 [dependencies.smallvec]
-version = "1.4"
+version = "1"
 
 [dependencies.sorted-iter]
 version = "0.1"
 [dev-dependencies.anyhow]
-version = "1.0.32"
-
-[dev-dependencies.criterion]
-version = "0.3.0"
+version = "1.0"
 
 [dev-dependencies.fnv]
 version = "1.0.7"
@@ -87,9 +84,6 @@
 [dev-dependencies.hex]
 version = "0.4.3"
 
-[dev-dependencies.hexdump]
-version = "0.1.1"
-
 [dev-dependencies.lazy_static]
 version = "1.4.0"
 
@@ -103,13 +97,13 @@
 version = "0.1.0"
 
 [dev-dependencies.quickcheck]
-version = "0.8"
+version = "1.0"
 
 [dev-dependencies.quickcheck_macros]
-version = "0.8.0"
+version = "1.0.0"
 
 [dev-dependencies.rand]
-version = "0.7.3"
+version = "0.8.5"
 
 [dev-dependencies.rkyv]
 version = "0.7.19"
@@ -118,12 +112,6 @@
 [dev-dependencies.serde_json]
 version = "1.0.41"
 
-[dev-dependencies.stats_alloc]
-version = "0.1.8"
-
-[dev-dependencies.testdrop]
-version = "0.1.2"
-
 [dev-dependencies.tokio]
 version = "1.13.0"
 features = ["full"]
